src/com/griddynamics/yvoronovskyi/battleship/BattleshipField.java

```diff
@@ -0,0 +1,228 @@
+            border.append(Constants.DELIMETER);
+            border.append(index);
+        }
+        border.append(Constants.LINE_SEPARATOR);
+        outStr.append(border);
+        for (int rowIndex = Constants.FIRST_INDEX; rowIndex < Constants.ROW_SIZE; rowIndex++) {
+            outStr.append(Character.toString(rowIndex + 65));
+            for (int columnIndex = Constants.FIRST_INDEX; columnIndex < Constants.COLUMN_SIZE; columnIndex++) {
+                outStr.append(Constants.DELIMETER);
+                String ch = fieldMap[rowIndex][columnIndex];
+                if (typePrint.equals(Constants.PRINT_SHOTS)) {
+                    ch = (ch.equals(Constants.SHIP)) ? Constants.EMPTY_CELL : ch;
+                }
+                outStr.append(ch);
+            }
+            outStr.append(Constants.LINE_SEPARATOR);
+        }
+
+        return outStr.toString();
+    }
+
+    public boolean isLoser() {
+        for (int rowIndex = Constants.FIRST_INDEX; rowIndex < Constants.ROW_SIZE; rowIndex++) {
+            for (int columnIndex = Constants.FIRST_INDEX; columnIndex < Constants.COLUMN_SIZE; columnIndex++) {
+                if (Constants.SHIP.equals(fieldMap[rowIndex][columnIndex])) {
+                    return false;
+                }
+            }
+        }
+        return true;
+    }
+
+    private boolean isOcupated(int x, int y) {
+        return (occupationMap[x][y].equals(Constants.EMPTY_CELL));
+    }
+
+    public boolean isShip(Point point) {
+        currentShipId = Constants.EMPTY_DELIMETER;
+        String ship = fieldMap[point.x][point.y];
+        if (ship.equals(Constants.SHIP)) {
+            currentShipId = shipIdMap[point.x][point.y];
+        }
+        return (ship.equals(Constants.SHIP) || ship.equals(Constants.HIT));
+    }
+
+    private boolean isShipAlive() {
+        for (int rowIndex = Constants.FIRST_INDEX; rowIndex < Constants.ROW_SIZE; rowIndex++) {
+            for (int columnIndex = Constants.FIRST_INDEX; columnIndex < Constants.COLUMN_SIZE; columnIndex++) {
+                if (currentShipId.equals(shipIdMap[rowIndex][columnIndex])) {
+                    return true;
+                }
+            }
+        }
+        return false;
+    }
+
+    public void makeFleet() {
+        System.out.println(this.printDesk(Constants.PRINT_ALL));
+
+        for (Map.Entry<String, Integer> entry : shipsFleet.entrySet()) {
+            makeShip(entry.getKey(), entry.getValue());
+        }
+    }
+
+    private void makeShip(String shipType, int desk) {
+        boolean isShipCoordinateValid = false;
+        System.out.println("Enter the coordinates of the " + shipType + "(" + desk + "cells):");
+        while (!isShipCoordinateValid) {
+            String coordinates = ConsoleReader.getString();
+            isShipCoordinateValid = Validator.validateCoordinates(coordinates);
+            if (!isShipCoordinateValid) {
+                System.out.println(Constants.COORDINATES_ERROR_MESSAGE);
```

Comment:
Please, change the name of constant to those that exactly identifies it, e.g. WRONG_COORDINATES_FORMAT

Reply:
Done.